### DISCUSSION TITLE
On php 8 I received pcntl error

### DISCUSSION THREAD

**Discussion by Ahmadrezagh** [spatie/laravel-backup#1381] [Q&A]
Unable to subscribe to signal events. Make sure that the `pcntl` extension is installed and that "pcntl_*" functions are not disabled by your php.ini's "disable_functions" directive.

**Comment by PaolaRuby** [comment]
Laravel versión? OS? Package versión? Symfony/Console version?
Make sure that the pcntl extension is installed and that "pcntl_*" functions are not disabled by your php.ini's "disable_functions" directivo.

I'am using php 8.0.9 without problems

**Comment by Ahmadrezagh** [comment]
Laravel : 8
Os : Ubuntu 20.04
Package version : ^7.6
I can't find Symfony/Console in my composer.json file 

this package work on my Macbook but when I upload it on server I received these errors

**Reply by PaolaRuby** [reply]
You can see the installed versions in composer.lock, not in composer.json
And what about this
"Make sure that the pcntl extension is installed and that "pcntl_*" functions are not disabled by your php.ini's "disable_functions" directivo."? Look at your php.ini
